itest/lnd_coop_close_rbf_test.go

```diff
@@ -0,0 +1,117 @@
+package itest
+
+import (
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
+	"github.com/stretchr/testify/require"
+)
+
+func testCoopCloseRbf(ht *lntest.HarnessTest) {
+	rbfCoopFlags := []string{"--protocol.rbf-coop-close"}
+
+	// Set the fee estimate to 1sat/vbyte. This ensures that our manually
+	// initiated RBF attempts will always be successful.
+	ht.SetFeeEstimate(250)
+	ht.SetFeeEstimateWithConf(250, 6)
+
+	// To kick things off, we'll create two new nodes, then fund them with
+	// enough coins for the test below.
+	alice := ht.NewNode("Carol", rbfCoopFlags)
+	defer ht.Shutdown(alice)
+
+	bob := ht.NewNode("Dave", rbfCoopFlags)
+	defer ht.Shutdown(bob)
```

Comment:
The test could be simplified a bit, also removes the pain from rebasing conflicts once the incoming itest PRs are merged,
```diff
diff --git a/itest/lnd_coop_close_rbf_test.go b/itest/lnd_coop_close_rbf_test.go
index c60e48541..0c61738ff 100644
--- a/itest/lnd_coop_close_rbf_test.go
+++ b/itest/lnd_coop_close_rbf_test.go
@@ -16,21 +16,16 @@ func testCoopCloseRbf(ht *lntest.HarnessTest) {
 	ht.SetFeeEstimateWithConf(250, 6)
 
 	// To kick things off, we'll create two new nodes, then fund them with
-	// enough coins for the test below.
-	alice := ht.NewNode("Carol", rbfCoopFlags)
-	defer ht.Shutdown(alice)
-
-	bob := ht.NewNode("Dave", rbfCoopFlags)
-	defer ht.Shutdown(bob)
-
-	// With the nodes active, we'll fund Alice's wallet, connect them, then
-	// fund enough coins to make a 50/50 channel.
-	ht.FundCoins(btcutil.SatoshiPerBitcoin, alice)
-	ht.ConnectNodes(alice, bob)
-	chanPoint := ht.OpenChannel(alice, bob, lntest.OpenChannelParams{
+	// enough coins for the test below, connect them, then fund enough
+	// coins to make a 50/50 channel.
+	cfgs := [][]string{rbfCoopFlags, rbfCoopFlags}
+	params := lntest.OpenChannelParams{
 		Amt:     btcutil.Amount(1000000),
 		PushAmt: btcutil.Amount(1000000 / 2),
-	})
+	}
+	chanPoints, nodes := ht.CreateSimpleNetwork(cfgs, params)
+	alice, bob := nodes[0], nodes[1]
+	chanPoint := chanPoints[0]
 
 	// Now that both sides are active with a funded channel, we can kick
 	// off the test.
@@ -113,5 +108,5 @@ func testCoopCloseRbf(ht *lntest.HarnessTest) {
 	aliceClosingTxid := ht.WaitForChannelCloseEvent(aliceCloseStream)
 	bobClosingTxid := ht.WaitForChannelCloseEvent(bobCloseStream)
 	require.Equal(ht, aliceClosingTxid, bobClosingTxid)
-	ht.Miner.AssertTxInBlock(block, aliceClosingTxid)
+	ht.AssertTxInBlock(block, aliceClosingTxid)
 }
```